packages/grafana-data/src/types/config.ts

```diff
@@ -135,7 +135,7 @@ export interface BootData {
   user: CurrentUserDTO;
   settings: GrafanaConfig;
   navTree: NavLinkDTO[];
-  themePaths: {
+  assets: {
```

Comment:
This is the detected change -- I think it is an OK change since it really is an internal detail:

<img width="635" alt="image" src="https://github.com/grafana/grafana/assets/705951/14fb6d3b-5fc6-498e-86e8-d9233a4bc7d9">

Reply:
This is fully exposed via `@grafana/runtime` so it is not as internal as we think

Reply:
Should be marked as breaking change since it's exported via `@grafana/runtime`.